lib/trino-record-decoder/src/main/java/io/trino/decoder/avro/AvroColumnDecoder.java

```diff
@@ -25,13 +25,16 @@
 import io.trino.spi.type.ArrayType;
 import io.trino.spi.type.BigintType;
```

Comment:
I saw that Kafka plugin writes its own `AvroRowEncoder` and actually Pulsar writes its own `AvroColumnDecoder ` in `pulsar-sql` now. The major concern can be since `PulsarAvroColumnDecoder` is derive from `AvroColumnDecoder`, we may want to keep their logic synced. All significant fields and methods in `AvroColumnDecoder` seems `private` though.